app/src/main/java/org/p2p/wallet/newsend/NewSendFragment.kt

```diff
@@ -33,9 +36,29 @@ class NewSendFragment :
     private val binding: FragmentSendNewBinding by viewBinding()
 
     override val presenter: NewSendContract.Presenter by inject()
+    private val glideManager: GlideManager by inject()
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         binding.toolbar.title = recipientUsername ?: recipientAddress
+        // TODO PWN-6090 make logic of setting token!
+        binding.widgetSendDetails.apply {
+            setSendLabel(getString(R.string.send_label))
```

Comment:
always is `send_label`, i think no need to change it in future

Reply:
yes but it is also not a UiKit part 

maybe we will need to make it with attrs

Reply:
hmm maybe

ok I will move it to UiKit